ovc/cloudspaces.go

```diff
@@ -186,13 +186,13 @@ func (s *CloudSpaceServiceOp) Create(cloudSpaceConfig *CloudSpaceConfig) (string
 
 // Delete a CloudSpace
 func (s *CloudSpaceServiceOp) Delete(cloudSpaceConfig *CloudSpaceDeleteConfig) error {
-	_, err := s.client.Post("/cloudapi/cloudspaces/delete", *cloudSpaceConfig)
+	_, err := s.client.Post("/cloudapi/cloudspaces/delete", *cloudSpaceConfig, DataActionTimeout)
```

Comment:
I would say this is an operation action.
Since cloudapi delete is not recursive meaning not allowed if there are still vms
This will delete the vfw only and update the model